ios/RNSScreenStack.m

```diff
@@ -507,6 +511,7 @@ - (void)dismissOnReload
   return nil;
 }
 
+#if !TARGET_OS_TV
```

Comment:
Should we eliminate whole `gestureRecognizerShouldBegin` method on TVOS? Is it never used there?

Reply:
I am probably misunderstanding something here, but I thought this was only used for the edge swipe recognisers defined in `setupGestureHandlers` which don't exist on tvOS. My Obj-C skills aren't the best, does removing this interact with something outside of `RNSScreenStack`?

Reply:
You can see here the implementation before `setupGestureHandlers` was introduced: https://github.com/software-mansion/react-native-screens/pull/761/files#diff-1c3e13a1b0dcbaa5f7cff5f3ce12cc915fb509725faf693fb65cd23ceb49c4a9L141, so I think we should do something like
```objc
#if TARGET_OS_TV
	return YES;
#else
	// rest of the code
```

and it should start here: https://github.com/software-mansion/react-native-screens/pull/761/files#diff-1c3e13a1b0dcbaa5f7cff5f3ce12cc915fb509725faf693fb65cd23ceb49c4a9R526, but I am not sure if this is 100% correct and if this method is ever fired on `tvOS`.

Reply:
If we do it like that, maybe we should also resolve using `simple_push` to default transition so we don't end up with some weird behaviors on `tvOS`. We could do that by returning `nil` for `tvOS` and `RNSScreenStackAnimationSimplePush` option here: `https://github.com/software-mansion/react-native-screens/blob/a6f91b6b7ef800f87c1ccc1dea95e1ebcb5604e7/ios/RNSScreenStack.m#L503`.